api/src/models/service_config/mod.rs

```diff
@@ -26,23 +26,26 @@
 use crate::models::service::ContainerType;
 use crate::models::Image;
 pub use environment::{Environment, EnvironmentVariable};
+use secstr::SecUtf8;
 use serde::Deserialize;
 use serde_value::Value;
 use std::collections::BTreeMap;
 use std::hash::Hash;
 use std::path::PathBuf;
-
 mod environment;
 mod templating;
 
-#[derive(Clone, Debug, Deserialize, Eq, Hash, PartialEq)]
+#[derive(Clone, Debug, Deserialize, Eq, PartialEq)]
 #[serde(rename_all = "camelCase")]
 pub struct ServiceConfig {
     service_name: String,
     image: Image,
     env: Option<Environment>,
-    // TODO: rename this field because it does not match to volumes any more (it is file content, cf. issue #8)
-    volumes: Option<BTreeMap<PathBuf, String>>,
+    // Renamed volume field to files to add more relevance
+    // Functionality will remain as-is
+    // Files will now be stored with improved security using SecStr
```

Comment:
I think this comment can be removed because that seems to be a note to yourself.

Reply:
Comment removed.